packages/runtime/container-runtime/src/summarizerTypes.ts

```diff
@@ -220,11 +266,14 @@ export interface ISummarizer
     run(onBehalfOf: string, options?: Readonly<Partial<ISummarizerOptions>>): Promise<void>;
     updateOnBehalfOf(onBehalfOf: string): void;
 
-    /** Attempts to generate a summary on demand. */
-    summarizeOnDemand(
-        reason: string,
-        options: Omit<ISubmitSummaryOptions, "summaryLogger">,
-    ): OnDemandSummarizeResult;
+    /** Attempts to generate a summary on demand. If already running, takes no action. */
+    summarizeOnDemand(options: IOnDemandSummarizeOptions): OnDemandSummarizeResult;
+    /**
+     * Enqueue an attempt to summarize after the specified sequence number.
+     * If afterSequenceNumber is provided, the summarize attempt is "enqueued"
+     * to run once an eligible op comes in with sequenceNumber \>= afterSequenceNumber.
```

Comment:
nit: remove "\" before >=

Reply:
The backslash `\` is required for the docs to compile. It escapes the `>` symbol.

Reply:
I didn't know that! :)